connectors/src/connectors/confluence/index.ts

```diff
@@ -237,50 +229,41 @@ export async function resumeConfluenceConnector(
 export async function cleanupConfluenceConnector(
   connectorId: ModelId
 ): Promise<Result<undefined, Error>> {
-  const connector = await ConnectorModel.findOne({
-    where: { type: "confluence", id: connectorId },
-  });
+  const connector = await ConnectorResource.fetchById(connectorId);
   if (!connector) {
     logger.error({ connectorId }, "Confluence connector not found.");
     return new Err(new Error("Connector not found"));
   }
 
-  return sequelizeConnection.transaction(async (transaction) => {
-    await Promise.all([
-      ConfluenceConfiguration.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluenceSpace.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluencePage.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-    ]);
+  await Promise.all([
+    ConfluenceConfiguration.destroy({
```

Comment:
what's the plan for these ? I suppose we likely want all these guys to get deleted at once from a `ConnectorResource` method as well ?

Reply:
Yes it's coming in follow up steps.